distribution/openrct2.d.ts

```diff
@@ -4363,6 +4363,21 @@ declare global {
         textAlign: TextAlignment;
     }
 
+    interface ProgressBarWidget extends WidgetBase {
+        colour: number;
+        /**
+         * The percentage of the bar to fill, ranging from 0-100 (inclusive).
+         */
+        percentage: number;
```

Comment:
For the API I would prefer a fraction between 0 and 1, but I see that it is a percentage internally, so I'm fine with it.

Reply:
I agree standardization is key here. The only existing use of percent in the api is for ride downtime, which is represented from 0-100. I'm in favor of keeping that structure for now.